plugins/module_utils/network/ios/config/prefix_lists/prefix_lists.py

```diff
@@ -149,6 +176,15 @@ def _compare(self, want, have):
                                     },
                                 )
                             else:
+                                if self.state == "merged" and have_prefix_param.get(
+                                    "sequence"
+                                ) == val.get(
+                                    "sequence"
+                                ):
+                                    self._module.fail_json(
+                                        "Merge state cannot merge on the pre-existing Prefix-List sequence,"
```

Comment:
Can we have the same failure message across all the platforms?
https://github.com/ansible-collections/cisco.nxos/pull/307/files#diff-3e585e63f353a34c61831886f8e47e3d9307a4b450ca47776a36d41c31039979R132-R134